src/state/SearchWordReducer.js

```diff
@@ -0,0 +1,10 @@
+const SearchWordReducer = (state, action) => {
+  switch (action.type) {
+    case 'SET_WORD':
+      return { ...state, word: action.payload };
+    default:
+      return state;
+  }
+};
```

Comment:
See how `payload` value is hold into `word` at the internal state of the reducer. So you might make easier reading on your code if you give meaningful names to your variables providede to the reducer users
```suggestion
const SearchWordReducer = (state, action) => {
  const { type, word } = action;
  switch (type) {
    case 'SET_WORD':
      return { ...state, word };
    default:
      return state;
  }
};
```